### ISSUE TITLE
Open folder in new window from browser window in spatial mode doesn’t work

### ISSUE THREAD

**Issue by scytho** [mate-desktop/caja#601]
Hi,
once there was a thing called spatial mode in Nautilus, which I found very useful when I used Ubuntu 10.04 Lucid Lynx (those were the days … best OS ever, everything just worked). Then the Gnome developers started Gnome 3 and decided “away with it”. Pity.

In Mate/Caja it’s still there, but it doesn’t work properly. The long title tells it: Open the Caja preferences, select “open every folder in its own window” (or something like that, I’m not using the english locale). Now, when you open a caja _browser_ window, and right-click on a folder and choose “open in a new window”, not “open in a new browser (folder?) window” … nothing happens. It should open the folder in a small window with no sidebar, no toolbar, no location bar, only a menu bar and the content of the folder.

**Comment by lukefromdc** [comment]
Spatial mode works fine, at least in GTK3 with the branch I'm using. To turn it on, fire up dcont-editor and go to mate>caja>preferences. The very first one listed is "always use browser" which defaults to true and prevents spatial windows from ever being used. Set it to "false" and restart Caja. Now caja opens everything in its own window with toolbars and sidebars hidden.

**Comment by scytho** [comment]
Yes, it does, you don’t even need the dconf-editor, you can check the option »open every folder in own window« or something like that in the preferences. The thing that _doesn’t_ work, is to open a folder in a spatial window out of a browser window using the right-click option.
